xbmc/games/dialogs/GUIDialogSelectGameClient.cpp

```diff
@@ -25,6 +25,7 @@
 #include "utils/URIUtils.h"
 #include "utils/log.h"
 
+using namespace ADDON;
 using namespace KODI;
 using namespace KODI::MESSAGING;
 using namespace GAME;
```

Comment:
The namespaces are starting to bring in a lot of symbols. I would prefer adding ADDON:: to the required symbols below.

Reply:
okay ;)

Reply:
Are you happy with this now @garbear?

Reply:
The diff now brings me joy. Thanks for namespacing all add-on stuff @howie-f !